aif360/metrics/mdss/ScoringFunctions/optim.py

```diff
@@ -0,0 +1,103 @@
+import numpy as np
+from aif360.metrics.mdss.ScoringFunctions.ScoringFunction import ScoringFunction
+
+
+def bisection_q_mle(score_function: ScoringFunction, observed_sum: float, probs: np.array, **kwargs):
+    """
+    Computes the q which maximizes score (q_mle).
```

Comment:
What is q?

Reply:
The alternative hypothesis which the MDSS optimization finds evidence for is that there is a multiplicative factor q in the odds for a subgroup of the data. For a subset of records, we can obtain the MLE of q (which is defined uniquely depending on the scoring function) and use this to score that subset.